server/main.go

```diff
@@ -98,13 +99,15 @@ func NameHandler(w http.ResponseWriter, r *http.Request) {
 
 	url := fmt.Sprintf("https://buildpack-registry.s3.amazonaws.com/buildpacks/%s.tgz", id)
 	log.Infof("at=download file=%s url=%s", shimmedBuildpack, url)
-	cmd := fmt.Sprintf(`curl --retry 3 --silent --location "%s" | tar xzm -C %s`, url, target_dir)
```

Comment:
retry is and redirect follow is built in to go's client, so we get these flags for free.